packages/cli/src/base.ts

```diff
@@ -46,6 +47,7 @@ export abstract class BaseCommand extends Command {
     if (this.requireSynced) {
       await requireNodeIsSynced(this.web3)
     }
+    addKeysToKit(this.kit, this.config.configDir)
```

Comment:
kit and web3 are lazy variables, they are only created if somebody access them. But adding this line, we automatically add them to all commands

i think is better to add this to `get kit()`